resources/js/Pages/App/Offer/OfferDetails.tsx

```
@@ -1,6 +1,6 @@
 import { usePage } from '@inertiajs/react'
 import type * as React from 'react'
-import { useEffect } from 'react'
+import { useEffect, useMemo } from 'react'
```

Biome (2.4.4)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)